src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -460,7 +488,11 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
       ReactFiberInstrumentation.debugTool.onWillBeginWork(workInProgress);
     }
     // See if beginning this work spawns more work.
+
+    priorityContextBeforeReconciliation = priorityContext;
```

Comment:
Hmm well it needs to change whenever `nextPriorityLevel` changes. Maybe in `findNextUnitOfWork`? I'll see.